server/server.go

```diff
@@ -230,7 +229,7 @@ func (s *Server) HTTPHandler() http.Handler {
 		)
 	}
 
-	handle("/api/tectonic/version", authHandler(s.versionHandler))
+	handle("/api/console/version", authHandler(s.versionHandler))
```

Comment:
This endpoint doesn't exist.

Reply:
I don't know what you mean by doesn't exist... Console serves it, although we don't currently use it in the UI.

Reply:
I tried to access `http://localhost:9000/api/console/version` and get `not found`.

Reply:
I changed the path in this pr

Reply:
Oh sorry, I'm dumb. I thought this was JS code.